File: python/prograimg_python_basic/Exercises/conditional_statements/conditional_statements_-_exercise/shopping.py
budget = float(input())
video_cards = int(input())
processors = int(input())
ram = int(input())
per = 0

if video_cards > processors:
    per = 0.15

video_price = 250
buy_cards = video_cards * video_price
processor_price = buy_cards * 0.35
ram_price = buy_cards * 0.10

buy_processor = processors * processor_price
buy_ram = ram * ram_price

end_price = (buy_ram + buy_processor + buy_cards) * per
end_sum = (buy_ram + buy_processor + buy_cards) - end_price

if end_sum > budget:
    need_more = end_sum - budget
    print("Not enough money! You need " + "{:.2f}".format(need_more) + " leva more!")

elif end_sum <= budget:
    left = budget - end_sum
    print("You have " + "{:.2f}".format(left) + " leva left!")

File: JS/JavaScript/JS-Basics/ConditionalStatementsExercise/Shopping.js
function shopping(inputs) {
    let budget = parseFloat(inputs[0]);
    let videos = Number(inputs[1]);
    let processors = Number(inputs[2]);
    let ram = Number(inputs[3]);

    let priceProcessor = videos * 250 * 0.35;
    let priceRam = videos * 250 * 0.10;

    let discount = 0;

    if (videos > processors) {
        discount = 0.15;
    }

    let price = videos * 250 + priceProcessor * processors + priceRam * ram;
    let endPrice = price - price * discount

    if (budget >= endPrice) {
        console.log(`You have ${(budget - endPrice).toFixed(2)} leva left!`);
    }
    else {
        console.log(`Not enough money! You need ${(endPrice - budget).toFixed(2)} leva more!`)
    }

}

shopping(["920.45",
"3",
"1",
"1"])
